scripts/apps/authoring/authoring/directives/AuthoringDirective.ts

```diff
@@ -713,7 +714,9 @@ export function AuthoringDirective(
             }
 
             $scope.showCustomButtons = function(item) {
-                if ($location.path() === '/workspace/personal') {
+                if ($scope.isPersonalSpace && appConfig?.features?.publishFromPersonal) {
```

Comment:
I was trying to see what custom buttons are. Looks like there are 4:

* To Desk
* Close & Continue
* Publish
* Publish & Continue

It would be good to clean up the logic of `showCustomButtons` to be something like

```
$scope.showCustomButtons = () => toDeskAvailable && closeAndContinueAvailable && publishAvailable;
```

It would also need some comments/improvement of variable names - it is not clear at all what "to desk" or "close and continue" means.